kitsune/search/v2/documents.py

```diff
@@ -137,12 +137,37 @@ def get_model(cls):
         return wiki_models.Document
 
 
+class UserInnerDoc(InnerDoc):
+    id = field.Integer()
+    username = field.Keyword()
+
+
+class ProductInnerDoc(InnerDoc):
+    id = field.Integer()
+    title = field.Keyword()
```

Comment:
Our product titles can be localised, should we index these at all, or index them as a multi-locale object?

Reply:
probably this makes sense.

Reply:
Shouldn't title be `Text` instead of `Keyword`?